src/pages/workshop-description/index.js

```diff
@@ -0,0 +1,232 @@
+      <Details>
+        <section className='description-section'>
+          <Link className="close-button" to={'/workshops'}>
+            <FontAwesomeIcon
+              icon={faRectangleXmark}
+              size="2xl"
+              style={{ color: 'black' }}
+            />
+          </Link>
+          <h1 className="title">{workshops.attributes?.nome}</h1>
+          <span className="span-detais">
+            <ul id="container-details">
+              <li>
+                <div className="start-date">
+                  <div className="spacingDate">
+                    <FontAwesomeIcon icon={faCalendarDays} size="lg" />
+                    <p>{formatWorkshopDates(workshops)}</p>
+                    <p>{formatWorkshopDuration(workshops)}</p>
+                  </div>
+                </div>
+              </li>
+              <li>
+                <div className="div-price">
+                  <p className="price">
+                    <FontAwesomeIcon
+                      icon={faHandHoldingDollar}
+                      size="lg"
+                      style={{
+                        '--fa-secondary-color': '#ffffff',
+                        '--fa-primary-opacity': '1'
+                      }}
+                    />
+                    <div className="spacingMoney">
+                      {workshops.attributes?.preco}
+                    </div>
+                  </p>
+                </div>
+              </li>
+              <li>
+                <div className="div-local">
+                  <p className="local">
+                    <div className="spacingLocal">
+                      <FontAwesomeIcon icon={faLocationDot} size="lg" />
+                      Oficina presencial em {workshops.attributes?.local}
+                    </div>
+                  </p>
+                </div>
+              </li>
+              <li>
+                <div className="workshopType">
+                  <p className="type">
+                    <FontAwesomeIcon icon={faBullhorn}
+                      style={{ '--fa-primary-color': '#000000', '--fa-secondary-color': '#000000', '--fa-secondary-opacity': '1', transform: 'rotate(-15deg)' }}
+                      size="lg"
+                    />
+                    <div className="spacingType">
+                      {workshops.attributes?.tipo}
+                    </div>
+                  </p>
+                </div>
+              </li>
+              <li>
+                <div className="partners">
+                  <p className="partner">
+                    <div className="parce">
+                      {workshops.attributes?.parceires?.data
+                        .filter(partner => partner !== null && partner !== undefined)
+                        .map((partner, index) => (<>
+                          <FontAwesomeIcon icon={faUser} size="lg" />
+                          <p key={index} className="spacing-partners">
+                            {partner.attributes?.nome}
```

Comment:
Se `attributes` for nulo ou undefined, vai mostrar coisa esquisita na tela. Nessas horas, o operador de null-coalescing é seu amiguinho para settar um valor default:

```suggestion
                            {partner.attributes?.nome ?? 'Sem Nome'}
```

Ou

```suggestion
                            {partner.attributes?.nome ?? '-'}
```

Ou qualquer outro valor que faça sentido aqui.